### ISSUE TITLE
`emoji` generic font-family

### ISSUE THREAD

**Issue by binyamin** [system-fonts/modern-font-stacks#7]
In the README, there is [a note about emoji support](https://github.com/system-fonts/modern-font-stacks#emoji-support).

Level 4 of the _CSS Fonts_ module defines an "emoji" generic font-family[^emoji]. I don't know how well OS's support it. It would be interesting to see if that could replace the current font-stack.

```css
html {
    font-family: system-ui, emoji;
}
```

[^emoji]: https://www.w3.org/TR/css-fonts-4/#valdef-font-family-emoji

**Comment by tedw** [comment]
FWIW here’s a discussion about emoji support https://github.com/necolas/normalize.css/issues/665#issuecomment-1116174450

**Comment by danklammer** [comment]
I love this idea of an `emoji` generic, but support isn't quite there yet.

We will wait this one out and hopefully add it to the [Emoji Support](https://github.com/system-fonts/modern-font-stacks#emoji-support) section one day.

**Comment by binyamin** [comment]
@danklammer Where did you find the support values? I had trouble finding them.

**Comment by danklammer** [comment]
@binyamin There isn't a lot on the topic. I referenced a couple articles and their comments:
- https://nolanlawson.com/2022/04/08/the-struggle-of-using-native-emoji-on-the-web/
- https://www.client9.com/css-color-emoji-stack/

I also tested myself using a fork of Nick Galbreath's CodePen:
https://codepen.io/danklammer/pen/QWVVVbx/86ca8d04b6fcb91ad3a9097f82ff486d?editors=1100&layout=left

From my understanding there is still a problem using the generic `emoji` for fonts that have their own symbols. That's where `font-variant: font-variant-emoji` will come in. Supposedly supported in Firefox but my CodePen didn't work for me on macOS on Firefox 111.0.1 (64-bit) at least when used with `system-ui`. I think it's just early.
https://caniuse.com/mdn-css_properties_font-variant_font-variant-emoji
